test/kubemark/start-kubemark.sh

```diff
@@ -227,7 +228,13 @@ function start-hollow-nodes {
 detect-project &> /dev/null
 create-kubemark-master
 
-MASTER_IP=$(grep server "$LOCAL_KUBECONFIG" | awk -F "/" '{print $3}')
+if [ -f "$INTERNAL_KUBECONFIG" ]; then
+    HOLLOWNODE_KUBECONFIG="$INTERNAL_KUBECONFIG"
```

Comment:
nit: can you please use ${var} instead of simply $var here and below?

Reply:
Sure, done.